README.md

```diff
@@ -72,7 +73,7 @@ The three generated files are the type, the actual implementation, and unit test
 
 ### Writing the Type
 
-The type contains the shape of your resources. The template renders the minimally necessary `name`, and `ensure` attributes. You can modify their description and the name's type to match your resource. Add more attributes as you see fit.
+The type contains the shape of your resources. The template provides the necessary `name` and `ensure` attributes. You can modify their description and the name's type to match your resource. Add more attributes as you wish.
```

Comment:
"... as you need."? Developers are pretty much on their own from here on, and need to make the changes they require for their goals.

We'll need to add more info on what/how they can do this later.